docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -3,23 +3,91 @@
 
-This article describes how to enable full disk access for both the Elastic Agent, required in order to enable Elastic Endpoint Security, and the legacy Endgame sensor.
+This article describes how to enable full disk access for the required security system extensions, the Elastic Endpoint sensor, and the legacy Endgame sensor.
+
+[discrete]
+[[macos-permissions]]
+== macOS permissions
+
+The behavior of the Endgame Sensor and Elastic Endpoint differs based on your macOS version. MDM/JAMF users can pre-approve all full disk access without granting permission to the sensors. However, non-MDM/JAMF users will be prompted or not prompted to enable full disk access for required security files depending on the macOS version and sensor type:
+
+**Endgame Sensor**
+
+- `10.13, 10.14, 10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds. 
+
+- `11.0` (Big Sur): Users cannot proceed with installation without first granting the sensor the ability to load <<system-extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the system extension. Upon approval, a second prompt appears to enable **Network Filtering**. Approve this final prompt for installation to proceed.
++
+You also must grant full disk access to `com.endgame.systemextension`
+
+- `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,esensor>> full disk access.
+
+
+**Elastic Endpoint**
+
+No prompts to approve the <<enable-kernel-extension,kernel extension>>, <<system-extension>>, or the <<endpoint-endgame-sensor,elastic-endgame>>, due to installation happening through the {agent}. After installation, Endpoint policies **will fail** to detect events until you approve and enable kernel or system extension loading and full disk access for each version, as reflected in the **Administration > Endpoints** page of the security application.
+
+- `10.13, 10.14, 10.15`: <<enable-kernel-extension,kernel extension>>.
+
+- `11.0` (Big Sur): <<system-extension>>, as well as `co.elastic.systemextension`.
+
+- - `10.14.6+, 10.15, 11.0`: <<endpoint-endgame-sensor,elastic-endgame>>
+
+[discrete]
+[[system-extension]]
+== System extension
+
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endpoint with system extensions, full-disk access must be enabled for the system extension during <<install-endpoint,installation>> on macOS Big Sur (11.0) and later.  
+
+--
+image::images/system-extension-prompt.png[System Extension Prompt]
+--
+
+If you select **Ok** and continue installation, you'll still need to enable Full Disk Access for the system extension.
```

Comment:
```suggestion
After you select **Ok** on macOS 11 you'll get a prompt to approve network content filtering, click Allow. Then enable Full Disk Access for the system extension.
```

Depending on the product the wording will vary slightly.

The Network Filtering pop up will look like this for Elastic Endpoint
<img width="263" alt="image" src="https://user-images.githubusercontent.com/56368752/98598394-66d4bc00-22a8-11eb-8fcb-d326e046dd90.png">


For Endgame Sensor it'll look like this
![image](https://user-images.githubusercontent.com/56368752/98598457-7c49e600-22a8-11eb-94e4-0b094a76a519.png)